score-mail/src/main/java/io/cloudslang/content/mail/entities/EncryptionAlgorithmsEnum.java

```diff
@@ -0,0 +1,40 @@
+package io.cloudslang.content.mail.entities;
+
+import org.apache.commons.lang3.StringUtils;
+import org.bouncycastle.mail.smime.SMIMEEnvelopedGenerator;
+
+/**
+ * Created by persdana on 8/3/2015.
+ */
+public enum EncryptionAlgorithmsEnum {
+    DES_EDE3_CBC(SMIMEEnvelopedGenerator.DES_EDE3_CBC), RC2_CBC(SMIMEEnvelopedGenerator.RC2_CBC),
+    IDEA_CBC(SMIMEEnvelopedGenerator.IDEA_CBC), CAST5_CBC(SMIMEEnvelopedGenerator.CAST5_CBC),
```

Comment:
For better visibility, I would recommend to aligned each enum values on a new line.